tests/Schema/Keywords/TypeTest.php

```diff
@@ -20,6 +20,7 @@ public function validDataProvider() : array
             ['string', null, 'string value'],
             ['object', null, ['a' => 1]],
             ['array', null, ['a', 'b']],
+            ['array', null, ['0' => 'a', '2' => 'b']], // this triggers: "Value expected to be 'array', 'array' given."
```

Comment:
actually `object given` should be raised here. but we have an issue about this already, I hope

Reply:
yea, this one https://github.com/thephpleague/openapi-psr7-validator/issues/25